packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -456,6 +476,7 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
       ),
       contextMenuLocation: _contextMenuLocation,
       previousChildRect: _decoyChildEndRect!,
+      scaleFactor: _scaleFactor,
```

Comment:
Why does the Route need to be scaled using the exact same scale factor? If the end goal is to make sure that everything is within the Safearea, shouldn't the Route calculate the scale factor using the size of its own content?

Reply:
sorry I misread this. Didn't realize ContextMenuRoute uses it to scale the size back.